way_to_home/utils/validators.py

```diff
@@ -245,4 +245,8 @@ def login_validator(data):
     required_keys = ['email', 'password']
     if not required_keys_validator(data, required_keys):
         return False
+    if not email_validator(data.get('email')):
+        return False
+    if not password_validator(data.get('password')):
+        return False
```

Comment:
You are sure that this is necessary for us? Therefore, there is no difference whether they are valid or not. During login, you only check if there is such data in the database. I think it is not necessary to validate data except data you will save in the database.